### PR TITLE
Update e107_admin/lancheck.php 

### PR DESCRIPTION
**Pull request by yesszus**
Button optimization

### CODE REVIEW

**Comment by yesszus** [comment]
Test this.
Before:
img

After:
img2

File: e107_admin/lancheck.php [hunk 1]
```diff
@@ -1383,7 +1383,7 @@ function check_core_lanfiles($checklan,$subdir='')
 
 				$editUrl = e_REQUEST_SELF."?".http_build_query($parms,'&amp;');
 				$text .="<td class='center' style='width:5%'>
-				<a href='".$editUrl."'  data-modal-caption='".$subpath."' class='e-modal btn btn-primary' type='button' style='width:60px'>".LAN_EDIT."</a>";
+				<a href='".$editUrl."'  data-modal-caption='".$subpath."' class='e-modal btn btn-primary' type='button' style='width:auto'>".LAN_EDIT."</a>";
 				$text .="</td></tr>";
```
Comment (CaMer0n) [inline e107_admin/lancheck.php]: Did you try just removing 'style' completely?

Comment (yesszus) [inline e107_admin/lancheck.php]: Ok, removing style and work for me.

**Comment by yesszus** [comment]
image

**Comment by yesszus** [comment]
Work this:

image

**Comment by yesszus** [comment]
@CaMer0n work this.

**Comment by CaMer0n** [comment]
Thank you @yesszus !

**Pull request merged**